scripts/cargo_metadata_dump.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python3
+
+from argparse import ArgumentParser
+from pathlib import Path
+import json
+import os
+import sys
+
+SCRIPT_DIR = os.path.dirname(os.path.abspath(__file__))
+sys.path.append(os.path.dirname(SCRIPT_DIR))
```

Comment:
Why is this needed? This script should be able to access local Python modules without needing to add its own directory into the `PYTHON_PATH`